README.MD

```diff
@@ -47,31 +47,31 @@ First we need to download these three Applications: [Hackintool](https://github.
 - Save the EDID as `Patched-EDID` or whatever name you like just to know which one is the patched one
 - Open the `Patched-EDID` with HexFiend and make sure you expand it so it contains 8 columns of code bytes and copy the 128 bytes of it.
 - Add child `AAPL00,override-no-connect` to `EFI>OC>Config.plist>DeviceProperties>PciRoot(0x0)/Pci(0x2,0x0)` with `Data` as the type and paste the 128 bytes of code in there
-- Save the config.plist file and reboot, Enjoy pure UEFI without a garbled screen.
+- Save the config.plist file, reboot and enjoy pure UEFI without a garbled screen.
 
-## GPU acceleration on Ventura and newer
+## Restoring GPU acceleration under Sonoma
 
-Since Apple dropped support for our SMBIOS (MacBookPro12,1) on Ventura and newer, the Broadwell graphic kexts have also been removed. This means we need to patch the Broadwell kexts back to Ventura and newer, which means we need to disable SIP. Disabling SIP results in losing the ability to apply Delta OTA updates. The patch must also be reapplied after each macOS update. To add the required kexts back to the system, you should use a tool called OpenCore Legacy Patcher, which has an amazing team that adds back support for older Macs.
+With Apple dropping support for all pre-Kaby Lake machines on Ventura and newer, graphics support for Broadwell and other previously supported GPUs has been removed as well. To restore GPU acceleration, we have to both weaken SIP and modify the root volume. Modifying the root volume results in losing the ability to apply Delta OTA updates. The root patches must be re-applied after each macOS update. To add the required kexts back to the system, use a tool called OpenCore Legacy Patcher, which has an amazing team that adds back support for older Macs.
 
-- Go to the latest OpenCore Legacy Patcher Release page: [Link](https://github.com/dortania/OpenCore-Legacy-Patcher/releases)
-- Download 'OpenCore-Patcher-GUI.app.zip'
-- After download, open the app and select 'Post Install Root Patch'.
+- Download the latest Sonoma development nightly branch of OpenCore Legacy Patcher: [Link](https://github.com/dortania/OpenCore-Legacy-Patcher/pull/1077#issuecomment-1646934494)
+- Open the application and select 'Post Install Root Patch'.
 - Wait for a while and reboot the system after it finishes.
-- Enjoy your E7250 on Sonoma :)
+- Replace `amfi=0x80` with `-amfipassbeta` under `boot-args` inside your config.plist to restore AMFI support.
```

Comment:
I don't think it's necessary to replace `amfi=0x80` since OCLP doesn't remove the boot-arg either. However, I think we can add `(Optional)` at the end instead.